platform/node/README.md

```diff
@@ -25,7 +25,6 @@ Further platforms might work [with additional libraries installed](https://githu
 
 ```
 npm test
-npm run test-suite
```

Comment:
This is a very old remnant. It has been years since Mapbox deleted this target from package.json.